src/Runner.Worker/StepsRunner.cs

```diff
@@ -118,7 +118,7 @@ public async Task RunAsync(IExecutionContext jobContext)
                 if (step is IActionRunner actionStep)
                 {
                     // Set GITHUB_ACTION
-                    step.ExecutionContext.SetGitHubContext("action", actionStep.Action.Name);
+                    step.ExecutionContext.SetGitHubContext("action", step.ExecutionContext.ContextName);
```

Comment:
i need to add a FF

Reply:
And when scopename not empty, need to set to `$"{step.ExecutionContext.ScopeName}.{step.ExecutionContext.ContextName}"`..... i might do that part after Ethan's PR lands for the CompositeActionOutputHandler changes